test/topology/test_topology_failure_recovery.py

```diff
@@ -4,73 +4,105 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Re commit message

> Closes https://github.com/scylladb/scylladb/issues/18614

Should be `Fixes`
in particular:
`Fixes scylladb/scylladb#18614`